apps/frontend/src/pages/access-management/access-management.tsx

```diff
@@ -44,13 +74,28 @@ const AccessManagement = (): JSX.Element => {
 		<PageLayout isLoading={isLoading}>
 			<h1 className={styles["title"]}>Access Management</h1>
 			<section>
-				<h2 className={styles["section-title"]}>Users</h2>
+				<div className={styles["section-header"]}>
+					<h2 className={styles["section-title"]}>Users</h2>
+				</div>
 				<Table<UserRow> columns={userColumns} data={userData} />
 			</section>
 			<section>
-				<h2 className={styles["section-title"]}>Groups</h2>
+				<div className={styles["section-header"]}>
+					<h2 className={styles["section-title"]}>Groups</h2>
+					<Button label="Create New" onClick={onModalOpen} />
+				</div>
 				<Table<GroupRow> columns={groupColumns} data={groupData} />
 			</section>
+			<Modal
```

Comment:
I don't know, but I have separated users and groups tables  into components and put modal there locally for groups. It's just an idea, I'm not sure about it, but you can think about it too.

Reply:
Yeah, it seems pretty logical. I will separate them. Thanks!